### PR TITLE
tiller fix

### PR DESCRIPTION
**Pull request by macsj200**
## PR Checklist

 - [ ] simple searchable title - `[mds-db] Add PG env var`, `[config] Fix eslint config`
 - [ ] briefly describe the changes in this PR
 - [ ] mark as draft if should not be merged
 - [ ] write tests for all new functionality

## Impacts
- [ ] Provider
- [ ] Agency
- [ ] Audit
- [ ] Policy
- [ ] Compliance
- [ ] Daily
- [ ] Native
- [ ] Policy Author

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -63,5 +63,8 @@
     "test:eslint": "yarn run:concurrent test:eslint",
     "test:unit": "yarn run:sequential test:unit",
     "values": "bin/mdsctl templateValues"
+  },
+  "dependencies": {
+    "chai": "4.2.0"
```
Comment (avatarneil) [inline package.json:68]: This should be up in `devDependencies`

Comment (macsj200) [reply package.json:68]: Yeah not sure why the deps even changed...

Comment (avatarneil) [reply package.json:68]: It's `mdsctl` -- it attempts to install `chai`. Not sure what line it is on in this PR, but on `develop` you can check out `bin/mdsctl::372`

**Comment by macsj200** [comment]
Abandoning

**Pull request closed without merging**